test/test-dumps-config

```diff
@@ -7,7 +7,8 @@ readonly image="$1"
 readonly series="$2"
 readonly cname="neo4j-$(uuidgen)"
 
-readonly dir=$(mktemp --directory)
+# Docker for mac doesn't allow access from /var/
```

Comment:
this comment is strange. you mean on Mac `mktemp` creates temporary files in `/var`?

Reply:
Yep:
```
mktemp --directory
/var/folders/wc/zvf2ggjj1x12lvbpqjl9cs3m0000gn/T/tmp.8Kge0Cklu0
```

Reply:
That's insane. How do you put up with these toy machines?

Reply:
@jgrobbel This comment and the one below are not sufficiently complete to be helpful. The pertinent in formation is that on OSX `mktemp` creates files under `/var` _and_ that that is not accessible from within Docker containers.

Reply:
Ack that.

You can configure it to work (which is more painful than just using the base config) but then there is a risk of mounting the hosting VM into the containers which will end badly.
https://github.com/docker/for-mac/issues/1532